app/src/main/java/com/infomaniak/mail/ui/main/menu/MoveViewModel.kt

```diff
@@ -49,20 +49,21 @@ class MoveViewModel @Inject constructor(
     private val messageUid inline get() = savedStateHandle.get<String?>(MoveFragmentArgs::messageUid.name)
     private val threadsUids inline get() = savedStateHandle.get<Array<String>>(MoveFragmentArgs::threadsUids.name)!!
 
+    var currentFolderId: String? = null
```

Comment:
Do we really want to rely on this kind of "side effect" for our logic? The value is not set as long as the getFolders() hasn't been called. This would become obvious when reading if the value was returned by the getFolders() instead. This would also prevent anyone from accessing it before it's ready

Reply:
Fixed here : https://github.com/Infomaniak/android-kMail/pull/1901